fair_logit_estimator.py

```diff
@@ -202,15 +202,10 @@ class FairLogitEstimator(BaseEstimator, ClassifierMixin):
         over sensitive attributes
     """
 
-    def fit(self, X, y, sensitive_col_idx, correlation_tolerance=None):
-        """A reference implementation of a fitting function
+    def __init__(self, sensitive_col_idx=0, correlation_tolerance=None):
```

Comment:
Also, since you are using a default for `correlation_tolerance`, the `sensitive_col_idx` parameter is redundant. You could require an 1d array of length n-features for `correlation_tolerance` using `-1` to denote a non-sensitive parameter.